lint-repository/action.yml

```
@@ -20,6 +20,12 @@ runs:
         fetch-depth: 0
         path: _repolint
 
-    - name: Run linter
+    - name: "check branch"
       shell: bash
-      run: repolinter lint _repolint
\ No newline at end of file
+      run: |
+          FILE=/home/runner/work/ae-trafficlight/ae-trafficlight/_repolint/repolint.json
+          if [ -f "$FILE" ]; then
+              repolinter lint _repolint
+          else 
+              repolinter lint _repolint --rulesetUrl https://raw.githubusercontent.com/simatic-ax/actions/refs/heads/test_repo_lint_fabian_unglert/lint-repository/repolint.json
+          fi
```

```suggestion
      run: |
          FILE=/home/runner/work/ae-trafficlight/ae-trafficlight/_repolint/repolint.json
          if [ -f "$FILE" ]; then
              echo "Execute the local settings file from the repository"
              repolinter lint _repolint
          else 
              echo "Execute the general settings"
              repolinter lint _repolint --rulesetUrl https://raw.githubusercontent.com/simatic-ax/actions/refs/heads/test_repo_lint_fabian_unglert/lint-repository/repolint.json
          fi
```

I like diagnostic :)